tests/queries/0_stateless/02494_query_cache_nested_query_bug.sh

```diff
@@ -15,7 +15,7 @@ ${CLICKHOUSE_CLIENT} --query "CREATE TABLE tab (a UInt64) ENGINE=MergeTree() ORD
 ${CLICKHOUSE_CLIENT} --query "INSERT INTO tab VALUES (1) (2) (3)"
 ${CLICKHOUSE_CLIENT} --query "INSERT INTO tab VALUES (3) (4) (5)"
 
-SETTINGS="SETTINGS use_query_cache=1, max_threads=1, allow_experimental_analyzer=0, merge_tree_read_split_ranges_into_intersecting_and_non_intersecting_injection_probability=0.0"
+SETTINGS="SETTINGS use_query_cache=1, max_threads=1, allow_experimental_analyzer=1, merge_tree_read_split_ranges_into_intersecting_and_non_intersecting_injection_probability=0.0"
```

Comment:
```suggestion
SETTINGS="SETTINGS use_query_cache=1, max_threads=1, merge_tree_read_split_ranges_into_intersecting_and_non_intersecting_injection_probability=0.0"
```
Maybe better to test with both analyzers?

Reply:
Yepp, will do that.